app/lib/tax_return_status.rb

```diff
@@ -43,18 +43,18 @@ def message_templates
 
       review_ready_for_qr: 301, review_reviewing: 302, review_ready_for_call: 303, review_signature_requested: 304, review_info_requested: 305,
 
-      file_ready_to_file: 401, file_efiled: 402, file_mailed: 403, file_rejected: 404, file_accepted: 405, file_not_filing: 406
+      file_ready_to_file: 401, file_efiled: 402, file_mailed: 403, file_rejected: 404, file_accepted: 405, file_not_filing: 406, file_hold: 407
   }.freeze
 
 
   ONBOARDING_STATUSES = [:intake_before_consent, :intake_in_progress, :intake_greeter_info_requested]
-  STATUS_KEYS_INCLUDED_IN_SLA = STATUSES.keys - [:intake_before_consent, :file_accepted, :file_not_filing].freeze
+  STATUS_KEYS_INCLUDED_IN_SLA = STATUSES.keys - [:intake_before_consent, :file_accepted, :file_not_filing, :file_hold].freeze
```

Comment:
I loved seeing file_hold added to `EXCLUDED_FROM_CAPACITY` and I wonder if it would also be helpful to create an `EXCLUDED_FROM_SLA`. Definitely not a blocker, just an observation.